Samples~/Audio/AudioSample.cs

```diff
@@ -11,25 +13,25 @@ public class AudioSample : MonoBehaviour
         [SerializeField] private AudioSource inputAudioSource;
         [SerializeField] private AudioSource outputAudioSource;
         [SerializeField] private Toggle toggleEnableMicrophone;
+        [SerializeField] private Dropdown dropdownAudioClips;
         [SerializeField] private Dropdown dropdownMicrophoneDevices;
+        [SerializeField] private Dropdown dropdownAudioCodecs;
+        [SerializeField] private Dropdown dropdownSpeakerMode;
         [SerializeField] private Button buttonStart;
         [SerializeField] private Button buttonCall;
         [SerializeField] private Button buttonHangup;
-        [SerializeField] private Text textChannelCount;
-        [SerializeField] private Text textMinFrequency;
-        [SerializeField] private Text textMaxFrequency;
-        [SerializeField] private AudioClip audioclipStereoSample;
+        [SerializeField] private AudioClip[] audioclipList;
```

Comment:
When I open this scene, I got error below.
I think some audioclip file is not commit to remote. Please check it.
```
MissingReferenceException: The variable audioclipList of AudioSample doesn't exist anymore.
You probably need to reassign the audioclipList variable of the 'AudioSample' script in the inspector.
UnityEngine.Object.get_name () (at /Users/bokken/buildslave/unity/build/Runtime/Export/Scripting/UnityEngineObject.bindings.cs:194)
Unity.WebRTC.AudioSample+<>c.<Start>b__21_0 (UnityEngine.AudioClip clip) (at Assets/Samples/Audio/AudioSample.cs:47)
System.Linq.Enumerable+SelectArrayIterator`2[TSource,TResult].ToList () (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
System.Linq.Enumerable.ToList[TSource] (System.Collections.Generic.IEnumerable`1[T] source) (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
Unity.WebRTC.AudioSample.Start () (at Assets/Samples/Audio/AudioSample.cs:46)
```

Reply:
fixed it